<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-14T16:35:07 IST" name="Default suite" finished-at="2021-12-14T16:35:08 IST" duration-ms="745">
    <groups>
    </groups>
    <test started-at="2021-12-14T16:35:07 IST" name="Default test" finished-at="2021-12-14T16:35:08 IST" duration-ms="745">
      <class name="com.te.bookmydoctor.resource.DoctorResourceEndPointTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.te.bookmydoctor.resource.DoctorResourceEndPointTest@3c73951]" started-at="2021-12-14T16:35:07 IST" name="beforeTest" finished-at="2021-12-14T16:35:08 IST" duration-ms="594" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="testSaveDoctor()[pri:1, instance:com.te.bookmydoctor.resource.DoctorResourceEndPointTest@3c73951]" started-at="2021-12-14T16:35:08 IST" name="testSaveDoctor" finished-at="2021-12-14T16:35:08 IST" duration-ms="26" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Syntax error, insert ";" to complete Statement
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Syntax error, insert ";" to complete Statement

at com.te.bookmydoctor.resource.DoctorResourceEndPointTest.testSaveDoctor(DoctorResourceEndPointTest.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSaveDoctor -->
      </class> <!-- com.te.bookmydoctor.resource.DoctorResourceEndPointTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
